client/my-sites/plugins/plugin-ratings/index.jsx

```diff
@@ -105,7 +113,10 @@ class PluginRatings extends Component {
 			<div className="plugin-ratings">
 				<div className="plugin-ratings__rating-stars">
 					<Rating rating={ rating } />
-					<span className="plugin-ratings__number">{ rating / 20 }</span>
+					{ inlineNumRatings && (
+						<span className="plugin-ratings__num-ratings">({ inlineNumRatings })</span>
+					) }
+					{ ! hideRatingValue && <span className="plugin-ratings__number">{ rating / 20 }</span> }
```

Comment:
Can we increase consistency by replacing `hideRatingValue` with `hideRatingNumber`?

Reply:
done 8284966f3a808426e9a78433dd75c7ee74df16d7